src/sentry/static/sentry/app/components/organizations/multipleProjectSelector.jsx

```diff
@@ -141,7 +145,20 @@ export default class MultipleProjectSelector extends React.PureComponent {
     } = this.props;
     const selectedProjectIds = new Set(value);
 
-    const allProjects = [...projects, ...nonMemberProjects];
+    const metaOptions = [];
+    if (multi) {
+      metaOptions.unshift({
+        id: null,
+        slug: t('My Projects'),
+      });
+    }
+    if (multi && organization.features.includes('open-membership')) {
```

Comment:
What about in the case where the user is in a role that has access to all projects?

Reply:
I forgot about that scenario. I'll add a check for organization.role as well. :+1: